doc/rados/operations/health-checks.rst

```diff
@@ -801,6 +801,92 @@ Or, to disable this alert on a specific OSD, run the following command:
 
    ceph config set osd.123 bluestore_warn_on_spurious_read_errors false
 
+BLOCK_DEVICE_STALLED_READ_ALERT
+_______________________________
+
+There are some logs found in OSD logging triggerred by BlueStore which can 
+cause bad disk. This can lead to degradation of performance in the cluster 
+before identifying the bad disk as the cause.
+
+``read stalled read 0x29f40370000~100000 (buffered) since 63410177.290546s, timeout is 5.000000s``
+
+However, this is difficult to spot as there's no discernible warning (a
+health warning or info in ``ceph health detail`` for example). More observations
+can be found here: https://tracker.ceph.com/issues/62500
+
+As there can be false positive ``stalled read`` indications, a mechanism
+has been added for more reliability. If in last ``bdev_stalled_read_warn_lifetime``
+duration ``stalled read`` indications are found more than or equal to
+``bdev_stalled_read_warn_threshold`` in block device of BlueStore, then this
+warning will be reported in ``ceph health detail``.
+
+By default value of ``bdev_stalled_read_warn_lifetime = 86400s`` and
+``bdev_stalled_read_warn_threshold = 1``. But user can configure it for
+individual OSDs.
+
+To change this, run the following command:
+
+.. prompt:: bash $
+
+   ceph config set osd.123 bdev_stalled_read_warn_lifetime 10
+
+.. prompt:: bash $
+
+   ceph config set osd.123 bdev_stalled_read_warn_threshold 5
```

Comment:
>run the following commands

Suggest placing both under the same `prompt` block, and defaulting to changing globally for all OSDs.  An admonition block can be added noting that this may be done surgically for individual OSDs or a given mask, say for only HDD OSDs.